lms/static/js/views/fields.js

```diff
@@ -491,7 +519,7 @@
                     placeholderValue: this.options.placeholderValue
                 }));
                 this.delegateEvents();
-                this.title((this.modelValue() || this.mode === 'edit') ? this.options.title : this.indicators['plus'] + this.options.title);
+                this.title((this.modelValue() || this.mode === 'edit') ? title : this.indicators.plus + title);
```

Comment:
Is this indicator HTML or plain text?  Does this need interpolateHtml or joinHtml?